From 4f7296c02d4580b29550f17ab62d9875ed75f485 Mon Sep 17 00:00:00 2001
From: Shunqian Zheng <zhengsq@rock-chips.com>
Date: Thu, 21 Jun 2018 18:17:17 +0800
Subject: [PATCH] ARM: rockchip_linux_defconfig: Add more functions for px3se

This mainly enables rk312x codec and inno mipi dphy.

Note that the ES8396 is used by px3se-evb board, and
SWITCH is required by rk312x codec.

Change-Id: Ie2a0715ba04a073875423329654fe3a77d46965d
Signed-off-by: Shunqian Zheng <zhengsq@rock-chips.com>
---
 arch/arm/configs/rockchip_linux_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/configs/rockchip_linux_defconfig b/arch/arm/configs/rockchip_linux_defconfig
index 4cbedbc4b466..9ad4bf75a2ce 100644
--- a/arch/arm/configs/rockchip_linux_defconfig
+++ b/arch/arm/configs/rockchip_linux_defconfig
@@ -350,6 +350,8 @@ CONFIG_SND_SOC_ROCKCHIP_MAX98090=y
 CONFIG_SND_SOC_ROCKCHIP_RT5645=y
 CONFIG_SND_SOC_CX20810=y
 CONFIG_SND_SOC_ES8323=y
+CONFIG_SND_SOC_ES8396=y
+CONFIG_SND_SOC_RK312X=y
 CONFIG_SND_SOC_RT5616=y
 CONFIG_SND_SOC_RT5640=y
 CONFIG_SND_SOC_SPDIF=y
@@ -418,6 +420,8 @@ CONFIG_LEDS_TRIGGER_TIMER=y
 CONFIG_LEDS_TRIGGER_ONESHOT=y
 CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
 CONFIG_LEDS_TRIGGER_MULTI_CTRL=y
+CONFIG_SWITCH=y
+CONFIG_SWITCH_GPIO=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_HYM8563=y
 CONFIG_RTC_DRV_RK808=y
@@ -456,6 +460,7 @@ CONFIG_PWM_ROCKCHIP=y
 CONFIG_PHY_ROCKCHIP_USB=y
 CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_DP=y
+CONFIG_PHY_ROCKCHIP_INNO_MIPI_DPHY=y
 CONFIG_PHY_ROCKCHIP_INNO_HDMI_PHY=y
 CONFIG_ANDROID=y
 CONFIG_NVMEM=y
-- 
2.35.3

